scalatest/src/main/scala/org/scalatest/matchers/MatchersHelper.scala

```diff
@@ -67,22 +67,26 @@ private[scalatest] object MatchersHelper {
     val methodNameStartsWithVowel = firstChar == 'a' || firstChar == 'e' || firstChar == 'i' ||
       firstChar == 'o' || firstChar == 'u'
 
-    def isFieldToAccess(field: Field): Boolean = field.getName == fieldNameToAccess
+    def isFieldToAccess(field: Field | Null): Boolean = field.nn.getName == fieldNameToAccess
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.